README.md

```diff
@@ -32,13 +32,16 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+* Bobby: I graduated from Miami University as a Software Engineer, but moved into a series of jobs where I worked as either an Infrastructure Engineer or, in the case of my last position, a DevOps Engineer. Unfortunately like all skills, when they aren't being actively used, they stagnate. I was being offered new positions and new fancy "Developer" titles, but at the end of the day, I couldn't help but feel as though I wasn't the true developer I wanted to be, and as each day went on-- my development skills were degrading. I enrolled in this program to help bolster those skills, and to bridge the gap between Academic programming, and Enterprise progamming.
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+* Bobby: I'd really love to go to Europe-- preferably somewhere like *Dublin* or *London*
```

Comment:
How about past travel?  Any place you've been before (either recently or a long time ago) that you enjoyed?  Please add a bit about any past travel destination (or the fact that you don't have one) to your branch and update your PR.

Reply:
Completed travel as asked for.